cookbooks/db-ssl/templates/default/remote_key_copy.sh.erb

```diff
@@ -17,15 +17,22 @@ function usage {
 }
 
 function push_keys {
+  local target=${1}
+
   if [[ "$user" != 'mysql' ]]
   then
-    eval ${ssh_options} ${instance} "mkdir -p ${user_home}/.${keyname}"
-    rsync -a -e "${ssh_options}" ${keyname}.crt ${instance}:${user_home}/.${keyname}/
-    rsync -a -e "${ssh_options}" ${keyname}.key ${instance}:${user_home}/.${keyname}/
-    rsync -a -e "${ssh_options}" root.crt ${instance}:${user_home}/.${keyname}/
+    eval ${ssh_options} ${target} "mkdir -p ${user_home}/.${keyname}"
+    rsync -a -e "${ssh_options}" ${keyname}.crt ${target}:${user_home}/.${keyname}/
+    rsync -a -e "${ssh_options}" ${keyname}.key ${target}:${user_home}/.${keyname}/
+    rsync -a -e "${ssh_options}" root.crt ${target}:${user_home}/.${keyname}/
   fi
 }
 
+function is_up() {
+  eval ${ssh_options} -o ConnectTimeout=10 $1 'date'
```

Comment:
This `-o ConnectTimeout=10` is redundant here, as it's already specified as part of `$ssh_options` above.

Reply:
This Looks Good To Me otherwise, although I haven't actually tested it myself!